documentation/example-file-usage/mqtt-connect-example-file-usage.md

```diff
@@ -0,0 +1,40 @@
+# Connecting to a M5 over CLI (via MQTT)
+
+This example file connects to your machine over MQTT to view basic data.
+
+1. Navigate to wherever you cloned this repository to. Open the "examples" folder and open a terminal window there, just like in the previous section.
+
+2. Type in the following command, but replace "YOUR_AUTH_TOKEN_HERE" with your actual Authentication Token and no quotes:
+
+   ```bash
+   python3 mqtt-connect.py -r us -A "YOUR_AUTH_TOKEN_HERE"
```

Comment:
Can we briefly explain `-r eu` vs `-r us`? Just to give people an idea that this is a critically important argument, that they *have* to care about.

Reply:
Ah yes, I totally overlooked that